### PR TITLE
task 7, updating the task to done

### PR DESCRIPTION
**Pull request by goldreb**

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -111,3 +112,43 @@ const showSuccess = (input) => {
...
+// console.log(taskCard);
+
+taskCard.addEventListener("click", (event) => {
+  event.preventDefault();
+
+  // if(event.target.classlist.contains("fa-square-check")){
+  //   const cardDone = event.target.parentElement.parentElement.parentElement;
+  //   console.log(cardDone)
+  // }
+
```
Comment (sfkacie) [inline assets/js/index.js:128]: If commented out, shall we erase?

File: assets/js/taskManager.js [hunk 2]
```diff
@@ -1,57 +1,120 @@
...
+
+      let taskHtml = createTaskHtml(
+        element.title,
+        element.details,
+        element.assignedTo,
+        element.status,
+        formattedDate,
+        element.id
+      );
+// console.log(taskHtml)
```
Comment (sfkacie) [inline assets/js/taskManager.js:83]: Comment need space before '//'?

File: assets/js/taskManager.js [hunk 3]
```diff
@@ -1,57 +1,120 @@
...
+//   deleteTask(taskId) {
+//     const newTasks = [];
+//     this.tasks.forEach((element) => {
+//       if (element.id !== taskId) {
+//         newTasks.push(element);
+//       }
+//     });
+//     this.tasks = newTasks;
+//   }
+
```
Comment (sfkacie) [inline assets/js/taskManager.js:118]: Delete if not needed?

**Review by sfkacie — APPROVED** [review]

File: assets/js/index.js (hunk 1 above)
Comment (goldreb) [reply assets/js/index.js:128]: yes you can erase that

File: assets/js/taskManager.js (hunk 3 above)
Comment (goldreb) [reply assets/js/taskManager.js:118]: yes

File: assets/js/taskManager.js (hunk 2 above)
Comment (goldreb) [reply assets/js/taskManager.js:83]: yes comment can do space as long as // type first

**Pull request merged**